swt/src/main/java/net/miginfocom/swt/SwtComponentWrapper.java

```diff
@@ -38,14 +38,22 @@
 import net.miginfocom.layout.LayoutUtil;
 import net.miginfocom.layout.PlatformDefaults;
 import org.eclipse.swt.SWT;
-import org.eclipse.swt.custom.StyledText;
 import org.eclipse.swt.graphics.*;
 import org.eclipse.swt.widgets.*;
 
 /**
  */
 public class SwtComponentWrapper implements ComponentWrapper
 {
+	private static Class<?> styledTextClz;
+
+	static {
+		try {
+			// Dynamically load StyledText because the RAP/RWT version of SWT doesn't have this class, so it may not exist
+			styledTextClz = Class.forName("org.eclipse.swt.custom.StyledText");
+		} catch (ClassNotFoundException ignored) {}

```

Comment:
Hm. Should we really completely ignore this exception? Or maybe put out some kind of warning somewhere. I understand why it is ignored ATM.

Reply:
I personally think it is fine. In case 1 (regular SWT), the class is found and loaded. If the class is actually used in the project it will work in the layout. In case 2 (RWT or some other SWT w/o the class), the class is not loaded. Since the class wasn't found, it could not possibly be used in the project either, so there are no corner cases where an error is silently ignored resulting in a broken layout.

That said, I have no real issue if we want to add a logger warning or something either, I just didn't see it as necessary.

Reply:
I understand. I simply have a fundamental problem with ignored exceptions... Except on Thread.sleep ;-)

Reply:
I'd like @mikaelgrev opinion prior to merging.

Reply:
For the reason nu11ptr brings up I think it is OK to swallow the exception here.